src/Azs.AzureBridge.Admin/test/Common.ps1

```diff
@@ -1,7 +1,8 @@
 $global:SkippedTests = @(
     "TestDownloadAzsAzureBridgeProductPipeline",
     "TestRemoveAzsAzureBridgeDownloadedProduct",
-    "TestRemoveAzsAzureBridgeDownloadedProductPipeline"
+    "TestRemoveAzsAzureBridgeDownloadedProductPipeline",
+    "TestGetAzsAzureBridgeProductByName"
```

Comment:
Can you please add comment why we are skipping them and what the plan to enable them back?

Reply:
I keep getting this error, I didn't know how to fix it and you said I can just skip it for now:

      ArgumentException: Missing Request 'Get-AzsAzureBridgeProduct+[NoContext]+TestGetAzsAzureBridgeProductByName+$GET+https://management.azure.com/subscriptions/74c72bdc-d917-431c-a377-8ca80f4238a0/resourceGroups/azurestack-activation/providers/Microsoft.AzureBridge.Admin/activations/default/products/microsoft.windowsserver2016datacenter-arm-2016.127.20171216?api-version=2016-01-01+1' in recording file
      Parameter name: rqKey
      at Get-AzsAzureBridgeProduct<Process>, D:\a\1\s\src\Azs.AzureBridge.Admin\exports\ProxyCmdletDefinitions.ps1: line 514
      at <ScriptBlock>, D:\a\1\s\src\Azs.AzureBridge.Admin\test\Get-AzsAzureBridgeProduct.Tests.ps1: line 67

Reply:
The recording for that test is missing.

Reply:
Please create an issue for this and assign to azurebridge team.

Reply:
I figure out it is just a bad error. The products don't actually exist, but the error says "unexpected response", when it should really say "a product by that name doesn't exist".

Here is the issue: https://github.com/Azure/azurestack-powershell/issues/16